src/terrain/terrain.cpp

```diff
@@ -588,43 +577,44 @@ Terrain::get_G_cost(const T tile, const Node<const T> node) {
     return node.get_time_cots() + get_H_cost(tile.sop(), node.get_tile()->sop());
 }
 
-void
-Terrain::add_all_adjacent(int xyz) {
-    tiles[xyz].clear_adjacent();
+int
+Terrain::pos(const NodeGroup* const node_group) const {
+    auto [x, y, z] = node_group->sop();
+    int px = floor(x) / Chunk::size;
```

Comment:
Is chunk::size a const? Why isn't it uppercased

Reply:
static const. were const things meant to be uppercase?

Reply:
1) If it's a class var it shouldn't be static
2) Make it a constexpr and uppercase it, you'd uppercase a `#define` right?